server/controllers/items/create.coffee

```diff
@@ -3,24 +3,32 @@ _ = __.require 'builders', 'utils'
 items_ = __.require 'controllers', 'items/lib/items'
 error_ = __.require 'lib', 'error/error'
 { Track } = __.require 'lib', 'track'
+{ Promise } = __.require 'lib', 'promises'
 snapshotEntityData = require './lib/snapshot/snapshot_entity_data'
 
 module.exports = (req, res, next)->
-  { body:item } = req
+  { body:items } = req
 
-  _.log item, 'item create'
+  singleItemMode = _.isPlainObject items
 
-  { entity:entityUri } = item
-  unless entityUri? then return error_.bundleMissingBody req, res, 'entity'
+  items = _.forceArray items
 
-  unless _.isEntityUri entityUri
-    return error_.bundleInvalid req, res, 'entity', entityUri
+  _.log items, 'create items'
+
+  for item in items
+    { entity:entityUri } = item
```

Comment:
new syntax for me.
meaning `entityUri = item[entity]` ?

Reply:
pretty much (`entityUri = item.entity` or `entityUri = item['entity']` to be exact)
It's part of the new ES6 [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Assigning_to_new_variable_names) capabilities that were ported back in CoffeeScript